tests/test_utils.hpp

```diff
@@ -7,7 +7,7 @@
 #include <catch2/matchers/catch_matchers_floating_point.hpp>
 #include <trigdx/reference.hpp>
 
-const size_t N = 1e7;
+const size_t N = 1e7 + 5;
```

Comment:
Why this random `5`? Can it be removed?
```suggestion
const size_t N = 1e7;
```